packages/react/src/component-tests/IcStepper/IcStepper.cy.tsx

```diff
@@ -0,0 +1,238 @@
+
+    cy.checkHydrated(STEPPER_SELECTOR);
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "full-width",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.015),
+    });
+  });
+
+  it("should render a left aligned stepper", () => {
+    mount(<LeftAligned />);
+
+    cy.checkHydrated(STEPPER_SELECTOR);
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "left-aligned",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.015),
+    });
+  });
+
+  it("should render a stepper with custom connector width", () => {
+    cy.window().then((win) => {
+      cy.stub(win, "ResizeObserver").callsFake((callback) => {
+        return {
+          observe: () => {
+            callback([{ contentRect: { width: 1024, height: 500 } }], this);
+          },
+          unobserve: () => null,
+          disconnect: () => null,
+        };
+      });
+    });
+
+    mount(<CustomConnectorWidth />);
+
+    cy.checkHydrated(STEPPER_SELECTOR);
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "custom-connector-width",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.05),
+    });
+  });
+
+  it("should render a custom stepper", () => {
+    cy.window().then((win) => {
+      cy.stub(win, "ResizeObserver").callsFake((callback) => {
+        return {
+          observe: () => {
+            callback([{ contentRect: { width: 1024, height: 500 } }], this);
+          },
+          unobserve: () => null,
+          disconnect: () => null,
+        };
+      });
+    });
+
+    mount(<Custom />);
+
+    cy.checkHydrated(STEPPER_SELECTOR);
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "custom",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.05),
+    });
+  });
+
+  it("should render a stepper without step titles", () => {
+    mount(<WithoutStepTitles />);
+
+    cy.checkHydrated(STEPPER_SELECTOR);
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "without-step-titles",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.001),
+    });
+  });
+
+  it("should render a stepper with hidden step information", () => {
+    cy.window().then((win) => {
+      cy.stub(win, "ResizeObserver").callsFake((callback) => {
+        return {
+          observe: () => {
+            callback([{ contentRect: { width: 1024, height: 500 } }], this);
+          },
+          unobserve: () => null,
+          disconnect: () => null,
+        };
+      });
+    });
+
+    mount(<WithHiddenStepInformation />);
+
+    cy.checkHydrated(STEPPER_SELECTOR);
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "with-hidden-step-information",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD),
+    });
+  });
+
+  it("should render a light stepper", () => {
+    mount(<Light />);
+
+    cy.checkHydrated(STEPPER_SELECTOR);
+
+    //cy.checkA11yWithWait();
```

Comment:
It could be, I've just left it commented out while there's a ticket open for the accessibility violation (I did the same for IcTabs and IcSearchBar). Would you prefer they were all removed?